services/net/notification/NotificationManager.cs

```diff
@@ -504,7 +504,7 @@ private async void SendOutNotificationEmailsAsync(IEnumerable<EmailContextModel>
                 // Save the notification instance.
                 var instance = new NotificationInstance(notification.Id, content.Id)
                 {
-                    Status = NotificationStatus.Completed,
+                    Status = NotificationStatus.Accepted,
```

Comment:
Notifications status will now reflect that CHES has accepted them, but not necessarily sent them.